.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question
+    url: https://join.slack.com/t/substra-workspace/shared_invite/zt-1fqnk0nw6-xoPwuLJ8dAPXThfyldX8yA
+    about: Don't hesitate to join the Substra community on Slack to ask all your questions!
+  - name: User Documentation Improvements
+    url: https://github.com/Substra/substra-documentation/issues
+    about: For issues related to the User Documentation, please open an issue on the substra-documentation repository
```

Comment:
Why having this specific part about the documentation? We could have one about the backend, substrafl, etc?

Reply:
Quite a lot of open source projects have a separate way of opening issues around documentation (it's not impacting the code, so it's less likely to introduce bugs, so it can be fast-forwarded by less experienced contributors). But then, Substra is probably not (yet ^^) at a size where this kind of things matters, so it might just be over-engineered. 
The other thing that was making it a special case in my opinion is that the Github link in the User Documentation leads to this repo, even though it's not the place where the documentation is living (and it is a bit disconcerting in my opinion). 
But then, I can also see the argument that we are centralising all issues in the substra/substra repo as an entry point, and dispatching them later on if needed.

Reply:
Related open question is: on which repos do we want to deploy this kind of issue template? All the public ones? Or do we want on the other hand to encourage users to open issues in one central place, because it can be hard to know for example if it's a bug in `substra` or `substrafl`?

Reply:
Thanks for the explanation:
- for the the link to the documentation, I don't have strong opinion I will let others comment
- I would deploy this to all the repos of the Substra org. Though the central one, i.e. the one to post where users don't know what is the buggy component should be `substra`. As we will merge `substra` and `substrafl`, substra repo will really become the "user facing" one.